app/build.gradle

```diff
@@ -5,11 +5,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 27
     defaultConfig {
         applicationId "ithaca_transit.android.cornellappdev.com.ithaca_transit"
         minSdkVersion 21
-        targetSdkVersion 26
+        targetSdkVersion 27
```

Comment:
Just checking, it's still compatible with API 21 devices and you've tested it on a device?